client/source/js/modules/model/calibration-ctrl.js

```diff
@@ -26,7 +26,7 @@ define(['./module', 'angular', 'underscore'], function (module, angular, _) {
 
     var initialize = function() {
```

Comment:
yes, it waits until the data is back from the sever before allowing the user to select automatic or manual calibration. I thought about removing it entirely, because nobody will do it. I still kept it as it felt the right thing to do.